src/api/index.ts

```diff
@@ -38,6 +38,7 @@ export type SchemaRegistryAPIClient = Client<{
     config: (_: any) => any
     updateConfig: (_: any) => any
     register: (_: any) => any
+    registered: (_: any) => any
```

Comment:
I cry a bit inside every time I see this 😅At least it's not exposed to the user 🤷